site/infra.ngs

```diff
@@ -58,5 +58,5 @@
     #            }
     #        ]
     # * Create CloudFront distribution pointing to each s3 public address, with alternate domain name (cname) `${REDIRECTS.id}.${REDIRECTS.host}`, use only North America and Europe, with redirect http to http, create ssl cert, TLSv1.2_2018, HTTP/2, no Default root object
-    # * Create DNS alias for all addressess that points to each cloudfront address
+    # * Create DNS alias for all addressees that points to each cloudfront address
```

Comment:
Typo in fixing typo. Should be `addresses`

Reply:
please do review it again. 

I apologize for the mistake.

Reply:
> please do review it again.

There is a button for that in the UI somewhere. Try to find it for next time.

> 
> I apologize for the mistake.

Completely optional step :)
It is assumed that people make mistakes.